README.md

```diff
@@ -22,6 +22,9 @@ Copacetic Action is supported with Copa version 0.3.0 and later.
 | --------------- | ------ | ------------------------------------ |
 | `patched-image` | String | Image reference of the patched image |
 
-## Example usage
+## Example Usage
 
-https://github.com/project-copacetic/copa-action/blob/941743581b0da5e581ca5a575f9316228c2f6c00/.github/workflows/patch.yaml#L1-L77
\ No newline at end of file
+https://github.com/project-copacetic/copa-action/blob/941743581b0da5e581ca5a575f9316228c2f6c00/.github/workflows/patch.yaml#L1-L77
+
+ **Patching Local Images** 
```

Comment:
i don't think we can add this due to action using buildkit containers directly (#32)